.spelling

```diff
@@ -1133,3 +1133,10 @@ enablement
 LUNs
 fileio
 iscsi-sbps-targets-config
+
+- operations/configuration_management/ARP_cache_tuning.md
+arp
+gc_thresh1
+gc_thresh2
+gc_thresh3
+CSM_RELEASE
```

Comment:
If all of these are truely wrapped with single quotes, then they'll pass the spellchecker and should be removed from `.spelling`.